.github/lint/.markdownlint.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Just wondering: Would it make sense to have that in the root directory so that it is found by VScode? Or does VScode automatically picks that up?
(We have other markdown files in the repo present...)

Reply:
I myself don't use VSCode. Quickly checking the documentation of the [markdownlint extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) would expect

`A custom configuration is often defined by a .markdownlint.json file in the root of the project`.

I don't have a strong opinion and the github action can easily be adapted, so I'm happy to move the file.

Should we also apply the linting to all markdown files in the repo for the github action?

Reply:
I just checked it in my VSCode - it already has something activated without having the specific config. I guess there are more things to do when we activate that for the whole repo. So no. We leave it that way for now.